odl/tomo/geometry/parallel.py

```diff
@@ -178,25 +178,36 @@ def __init__(self, apart, dpart, **kwargs):
             ``det_init_pos`` is used, which is only valid if
             ``det_init_axis`` is not zero.
         """
-        det_init_pos = kwargs.pop('det_init_pos', (1.0, 0.0))
-        det_init_axis = kwargs.pop('det_init_axis', None)
+        self._det_init_pos = kwargs.pop('det_init_pos', (1.0, 0.0))
+        self._det_init_axis = kwargs.pop('det_init_axis', None)
 
-        if det_init_axis is None:
-            if np.linalg.norm(det_init_pos) <= 1e-10:
+        if self.det_init_axis is None:
+            if np.linalg.norm(self.det_init_pos) <= 1e-10:
                 raise ValueError('initial detector position {} is close to '
                                  'zero. This is only allowed for explicit '
-                                 'det_init_axis.'.format(det_init_pos))
+                                 'det_init_axis.'
+                                 ''.format(self.det_init_pos))
 
-            det_init_axis = perpendicular_vector(det_init_pos)
+            det_init_axis = perpendicular_vector(self.det_init_pos)
 
         detector = Flat1dDetector(part=dpart, axis=det_init_axis)
         super().__init__(ndim=2, apart=apart, detector=detector,
-                         det_init_pos=det_init_pos)
+                         det_init_pos=self.det_init_pos)
 
         if self.motion_partition.ndim != 1:
             raise ValueError('angle partition has dimension {}, expected 1.'
                              ''.format(self.motion_partition.ndim))
 
+    @property
+    def det_init_pos(self):
+        """The position of the detector reference point at angle=0"""
```

Comment:
Side remark: we seem to tend towards the angle=0 solution to the problem raised in #295. Shall we definitely settle for that?